config/container.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace OWC\Zaaksysteem;
+
+use DI\Container;
+
+/**
+ * Link interfaces to their concretions.
+ */
+
+return [
+    /**
+     * Set the API variables and credentials
+     * @todo overwrite this through admin settings.
+     */
+    'api_uri'           => '',
```

Comment:
Does this mean the Gutenberg blocks `Zaken` and `Mijn Zaken` play different roles? the first is loading the settings from Gravity Forms and the second looks at this configuration file?

I would like to discuss this in a moment.